CONTRIBUTING.md

```diff
@@ -36,15 +36,15 @@ Please create a new issue on [GitHub](https://github.com/climate-processes/tobac
 * Before you start a pull request, please make sure that you added [numpydoc docstrings](#docstringExample) to your functions. This way the api documentation will be parsed properly.
 * If it is a larger change or an newly added feature or workflow, please place an example of use in the [tobac-tutorials](https://github.com/climate-processes/tobac-tutorials) repository or adapt the existing examples there.
 * If necessary add a folder or modify a file.
-
+* The code should be PEP 8 compliant, as this facilitates our collaboration. Please use the latest version of [black](https://black.readthedocs.io/en/stable/) to format your code. When you submit a pull request, all files are checked for formatting.
```

Comment:
In the current PR, the code is only checked for compliance and not formatted directly. I agree, that an auto-formatting of pull requests would be practical, but a caveat is that such action would change the code in the same commit, I think. The history is then much more difficult to comprehend,since the made changes are not highlighted anymore (as you can see in my formatting PR). Maybe, it is possible to run an auto-formatting and change the PR in a new commit, but I guess than it could be problematic with pull requests from private forks....I can have a look at that again.

Reply:
I had a look into this, unfortunately you can't commit to a PR using github actions as it's a security issue. There is the possibility that we could set up a bot account to do this with maintainer status, but I'm not sure what that would involve and it's probably quite a bit of work for not a huge benefit!

Reply:
Thank you for looking into this @w-k-jones! My personal opinion would be that it is OK to require that contributors do the formatting themselves and run black on the files they have changed.